src/node.cpp

```diff
@@ -85,19 +85,27 @@ void publish_scan(ros::Publisher *pub,
     if (!inverted) { // assumes scan window at the top
         for (size_t i = 0; i < node_count; i++) {
             float read_value = (float) nodes[i].distance_q2/4.0f/1000;
+            float range = read_value * distance_factor;
+            if (range < scan_msg.range_min)
+                range = std::numeric_limits<float>::infinity();
```

Comment:
Well, it's more like either ignore infs or use then as proof that there's free space up until max range. We've set things to ignore right now b/c an rplidar inf isn't meaningful.